site/js/index.js

```
@@ -1,223 +1,223 @@
-$("#no-script").hide();
-$("#loading").css('display', 'flex');
+$('#no-script').hide();
+$('#loading').css('display', 'flex');
 
-String.prototype.contains = function(substr) {
-    return this.toLowerCase().includes(substr.toLowerCase())
-}
-String.prototype.isJSON = function() {
+String.prototype.contains = function (substr) {
+  return this.toLowerCase().includes(substr.toLowerCase());
+};
```

ESLint

[error] String prototype is read only, properties should not be added.

(no-extend-native)